README.md

```diff
@@ -23,7 +23,7 @@ Shopify’s TSLint rules come bundled in `tslint-config-shopify`.
 To enable these rules, create a `tslint.json` file at the root level of your project, and extend `tslint-config-shopify`.
 ```
 {
-  "extends": "tslint-config-shopify"
+  "extends": "tslint-config-shopify/base"
```

Comment:
Thinking about it some more, it makes sense to make people explicitly opt out of typed rules.  i.e.,:
1. The default `tslint-config-shopify` export is a very opinionated set of Shopify's rules (typed included)
2. There's a separate `tslint-config-shopify/react` export (à la https://github.com/Shopify/eslint-plugin-shopify/blob/master/index.js#L16)
3. ... the react config extends the base config (à la https://github.com/Shopify/eslint-plugin-shopify/blob/master/lib/config/react.js#L4)
4. Then we have a separate `tslint-config-shopify/untyped` config that extends the base config, but disables all typed rules

Then a consuming project can have:
- A `tslint-full.json` that extends `tslint-config-shopify` (or `tslint-config-shopify/react`), 
- A `tslint.json` that does:
```json
{
  "extends": [
    "./config/tslint/full.json",
    "tslint-config-shopify/untyped"
  ]
}
```